some minor updates

thezoggy opened the pull request:
Tweaked the lang regex to catch some things we missed. Still needs to be tested against consoles/ebooks/etc. Left the music regex alone for now, looks like it should just be merged to the abbrv regex since its so similar. Updated the abbrv regex with some other languages that I've seen stuff released for.

thezoggy committed:
Tweaked the first few regexes to catch some things we missed. Added other 2 letter countries that I've seen stuff released for... for now I left the music one alone but I think we could just merge it with the abbrv one since it overlaps so much.

bruvv commented:
Nice update, you should also checkout: 
https://github.com/kevinlekiller/Newznab-Blacklist/issues/11

Maybe you can rewrite them.

thezoggy committed:
Tweaked the lang regex to catch some things we missed.
  > Still needs to be tested against consoles/ebooks/etc. Left the music regex alone for now, looks like it should just be merged to the abbrv regex since its so similar.

* Updated the abbrv regex with some other languages that I've seen stuff released for (Remove NZ from country as they speak english... so technically we shouldn't remove them (as we focus on non-english))

* Catch `VOST` when used as a release group

thezoggy commented:
tweaked some things, still trying to work out why i cant do a look-behind match to rule out some things..
